DOC-1091 Intrepid release notes (#1738)
RedisLabs/redislabs-docs: content/kubernetes/release-notes/k8s-6-2-8-10-2022-01.md

---
Title: Redis Enterprise for Kubernetes Release Notes 6.2.8-10 (January 2022)
linktitle: 6.2.8-10 (January 2022)
description: Support for Istio as ingress controller, K8s 1.22 (AKS, kOps, GKE), OpenShift 4.9. 
weight: 66
alwaysopen: false
categories: ["Platforms"]
aliases: [
    /kubernetes/release-notes/k8s-6-2-8-10-2022-01.md,
    /kubernetes/release-notes/k8s-6-2-8-10-2022-01/,   ]
---
## Overview

> The Redis Enterprise [K8s 6.2.8-10](https://github.com/RedisLabs/redis-enterprise-k8s-docs/releases/tag/v6.2.8-10) release provides support for the [Redis Enterprise Software release 6.2.8]({{<relref "/rs/release-notes/rs-6-2-8-october-2021.md">}}) and includes several enhancements and bug fixes.

Reviewer: The link [K8s 6.2.8-10](https://github.com/RedisLabs/redis-enterprise-k8s-docs/releases/tag/v6.2.8-10) gives me a 404.

Author: This will be resolved when @amiramm creates the release tag on their github project.

The key new features, bug fixes, and known limitations are described below.

## Images

This release includes the following container images:

* **Redis Enterprise**: `redislabs/redis:6.2.8-53` or  `redislabs/redis:6.2.8-53.rhel7-openshift`
* **Operator**: `redislabs/operator:6.2.8-10`
* **Services Rigger**: `redislabs/k8s-controller:6.2.8-10` or `redislabs/services-manager:6.2.8-10` (on the Red Hat registry)

## Feature improvements

> * Istio gateway/virtual services are supported as ingress controllers. Note that for Active-Active databases, the operator doesn't create ingress rules, and those should be manually configured (RED-64020)

Reviewer 2: I updated the document in Google docs - mention here that Istio version 1.12.0 was tested.

In OLM-deployed operators, the deployment of the cluster will fail if the name is not "rec". When the operator is deployed via the OLM, the security context constraints (scc) are bound to a specific service account name (i.e., "rec"). The workaround is to name the cluster "rec".

### REC clusters fail to start on Kubernetes clusters with unsynchronized clocks (RED-47254)

When REC clusters are deployed on Kubernetes clusters with unsynchronized clocks, the REC cluster does not start correctly. The fix is to use NTP to synchronize the underlying K8s nodes.

### Deleting an OpenShift project with an REC deployed may hang (RED-47192)

When a REC cluster is deployed in a project (namespace) and has REDB resources, the
REDB resources must be deleted first before the REC can be deleted. As such, until the
REDB resources are deleted, the project deletion will hang. The fix is to delete the
REDB resources first and the REC second. Afterwards, you may delete the project.

### REC extraLabels are not applied to PVCs on K8s versions 1.15 or older (RED-51921)

In K8s 1.15 or older, the PVC labels come from the match selectors and not the

> PVC templates. As such, these versions can not support PVC labels. If this feature

Reviewer: ```suggestion
PVC templates. As such, these versions cannot support PVC labels. If this feature
```
